### PR TITLE
webkit2gtk_4_1 => 2.44.3-icu75.1 

### PR DESCRIPTION
**Pull request by satmandu**
May help with #10401 

Builds:
- [x] `x86_64`
- [ ] `i686`
- [x] `armv7l`
##
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/satmandu/chromebrew.git CREW_BRANCH=webkit crew update \
&& yes | crew upgrade
```

### CODE REVIEW

**Review by uberhacker — APPROVED** [review]

**Comment by uberhacker** [comment]
@satmandu: You will need to merge master into your working branch to avoid failing tests.

File: packages/webkit2gtk_4_1.rb [hunk 1]
```diff
@@ -77,6 +76,10 @@ class Webkit2gtk_4_1 < Package
   no_env_options
 
   def self.patch
+    # Fix inconpatibility with gtk3 from current gobjects_introspection causing buuld failure
```
Comment (Zopolis4) [inline packages/webkit2gtk_4_1.rb:79]:
```suggestion
    # Fix incompatibility with gtk3 from current gobjects_introspection causing build failure
```

**Review by Zopolis4 — CHANGES_REQUESTED** [review]
nitpicking, but

File: packages/webkit2gtk_4_1.rb [hunk 2]
```diff
@@ -3,18 +3,17 @@
 class Webkit2gtk_4_1 < Package
   description 'Web content engine for GTK'
   homepage 'https://webkitgtk.org'
-  version '2.44.2'
+  version '2.44.3-icu75.1'
   license 'LGPL-2+ and BSD-2'
   compatibility 'x86_64 aarch64 armv7l'
-  min_glibc '2.37'
```
Comment (Zopolis4) [inline packages/webkit2gtk_4_1.rb:9]: yay!

**Review by Zopolis4 — APPROVED** [review]

**Pull request merged**